### ISSUE TITLE
Is this extension currently functioning?

### ISSUE THREAD

**Issue by kmgallahan** [mrlacey/CollapseComments#8]
### Installed product versions
- Visual Studio: 2019
- This extension: 1.1

### Description
Does the latest version of this extension work with the latest version of Visual Studio?

I haven't used the extension before, and after installing it the hotkey command does not appear to do anything. Just wanted to do a sanity check.

**Comment by mrlacey** [comment]
@kmgallahan I've just re-verified on versions 16.4.2 & 16.5.0 preview 1.0 and it works there fine.

What project type are you using and what is the extension of the file you have open?

**Comment by kmgallahan** [comment]
I've just tried:

- Creating a new project using the current C# unit test template (.cs)
- Added a comment to both test methods
- Tried the default Collapse Comment hotkeys (nothing)
- Tried a new hotkey combination (nothing)
- Tried CTRL+M, CTRL+O to see if that worked for collapsing all definitions (it did)

I'm not sure if this extension relies on code outlining to be enabled or not, but as mentioned I did ensure it was on for testing.

```
using NUnit.Framework;

namespace Sometests.Tests
{
    public class Tests
    {
        /* This is a test of the public broadcast system.
         This is a test of the public broadcast system.
         This is a test of the public broadcast system.*/
         [SetUp]
        public void Setup()
        {
        }

        // This is a test of the public broadcast system.
        // This is a test of the public broadcast system.
        // This is a test of the public broadcast system.
        [Test]
        public void Test1()
        {
            Assert.Pass();
        }
    }
}
```

**Comment by mrlacey** [comment]
That should work fine.

![image](https://user-images.githubusercontent.com/189547/71619354-f35c1b80-2bbb-11ea-9d58-e9b1f0f7215d.png)

Check the shortcut configuration.
Go to **Tools > Options... > Environment > Keyboard**, then search for "Edit.CollapseComments" to check it has a shortcut defined (installing the extension should have created this.)

![image](https://user-images.githubusercontent.com/189547/71619331-da536a80-2bbb-11ea-80f4-a808b0ea66d0.png)

Also check if the `Ctrl+M, Ctrl+C` command is also being used by something else by putting focus in the "Press shortcut keys:" text box and then pressing the key commands. Then check the drop down to see if it is conflicting with another command (which may stop either of them running.)
Try assigning a new shortcut and see if that works.

**Comment by kmgallahan** [comment]
As mentioned, already tried a new key combination without any luck. Also ensured nothing else was using the original or new hotkeys.

Since it works in your environment I'll just assume something is off about mine and if I spent enough time investigating it could be resolved.

I actually prefer to have outlining disabled as I find it fairly distracting. To that end, I did find an alternative comment hiding extension that works without outlining: [NoComment](https://marketplace.visualstudio.com/items?itemName=3dGrabber.NoComment)

It uses deprecated VS APIs so the installation isn't smooth, but otherwise it works well.
